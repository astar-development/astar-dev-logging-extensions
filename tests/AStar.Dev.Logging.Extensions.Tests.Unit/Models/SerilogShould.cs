using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Logging.Extensions.Models;

[TestSubject(typeof(Serilog))]
public class SerilogShould
{
    [Fact]
    public void ContainTheExpectedProperties()
        => new Serilog().ToJson().ShouldMatchApproved();
}