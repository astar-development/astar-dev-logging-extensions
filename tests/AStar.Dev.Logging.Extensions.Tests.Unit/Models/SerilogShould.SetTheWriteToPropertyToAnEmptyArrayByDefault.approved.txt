[
  {
    "name": "Seq",
    "args": {
      "serverUrl": "http://seq:5341"
    }
  }
]